### PR TITLE
[1530] simple well activity report history (starting point)

### PR DESCRIPTION
**Pull request by stephenhillier**

### CODE REVIEW

**Comment by stephenhillier** [comment]
[EPIC#Edit | Access original submissions |8| !!! [gherkintodo] [BV: 27]](https://trello.com/c/NygEYz2v/1451-epicedit-access-original-submissions-8-gherkintodo-bv-27)

File: app/backend/wells/views.py [hunk 1]
```diff
@@ -256,6 +259,23 @@ class WellTagSearchAPIView(ListAPIView):
     )
 
 
+class WellSubmissionsListAPIView(ListAPIView):
```
Comment (Sybrand) [inline app/backend/wells/views.py]: Many ways to skin a cat - but you could also leverage of a more generic approach (see https://www.django-rest-framework.org/api-guide/filtering/#specifying-a-default-ordering)

File: app/frontend/src/submissions/views/SubmissionDetail.vue [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+  },
+  computed: {
+    ...mapGetters(['codes', 'userRoles'])
+  },
+  methods: {
+    fetchSubmission () {
+      ApiService.get('submissions', this.$route.params.submissionId).then((response) => {
+        this.submission = response.data
+      }).catch((e) => {
+
```
Comment (Sybrand) [inline app/frontend/src/submissions/views/SubmissionDetail.vue]: Please do something here - at least like a console.error(e) - empty catch statements make things hard to debug.

**Comment by stephenhillier** [comment]
[EPIC#Edit | Access original submissions |8| !! [BV: 27]](https://trello.com/c/NygEYz2v/1451-epicedit-access-original-submissions-8-bv-27)

**Comment by stephenhillier** [comment]
[EPIC#Edit | Implement Change History |5| !!!](https://trello.com/c/oArW0sYg/374-epicedit-implement-change-history-5)

**Comment by stephenhillier** [comment]
[EPIC#Edit | Implement Change History |5| !!!](https://trello.com/c/oArW0sYg/374-epicedit-implement-change-history-5)

File: app/backend/gwells/change_history.py [hunk 3]
```diff
@@ -77,6 +77,8 @@ def generate_history_diff(collection, identifier=None):
         if identifier:
             item["name"] = identifier
 
+        print("user is", cur['update_user'], "create_user is", cur['create_user'])
```
Comment (Sybrand) [inline app/backend/gwells/change_history.py]:
```suggestion
```

**Pull request merged**